src/credentials/loginManager.ts

```diff
@@ -69,15 +69,13 @@ export class LoginManager {
         } catch (err) {
             // TODO: don't hardcode logic using error message, have a 'type' field instead
             if (!(err as Error).message.includes('cancel')) {
-                notifyUserInvalidCredentials(args.providerId)
-                getLogger().error(
-                    `Error trying to connect to AWS with Credentials Provider ${asString(
-                        args.providerId
-                    )}. Toolkit will now disconnect from AWS. %O`,
-                    err as Error
-                )
+                const msg = `login: failed to connect with "${asString(args.providerId)}": ${(err as Error).message}`
+                if (!args.passive) {
+                    notifyUserInvalidCredentials(args.providerId)
+                    getLogger().error(msg)
```

Comment:
Skip user-facing message _and_ logging during autoconnect. Because:

1. autoconnect (the caller) already logs a warning
1. for troubleshooting, the full details of a particular failed profile can always be found by doing an explicit login

Reply:
Makes sense to me. Overtime we'll want to start cutting back on what we classify as 'errors' anyway. Otherwise the logs will be noisy.